public/style.css

```diff
@@ -0,0 +1,4 @@
+body{
```

Comment:
This should not be in the `public` folder. I would make a `src` folder in the root, and have the `main.css` (or whatever you want to name the file) in there, along with the `js` folder.